o2a/mappers/prepare_mixin.py

```diff
@@ -13,38 +13,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Prepare node mixin"""
-from typing import Dict, List, Tuple
-import xml.etree.ElementTree as ET
+from typing import List, Tuple, Optional
 
+from o2a.converter.task import Task
+from o2a.o2a_libs.property_utils import PropertySet
 from o2a.utils import xml_utils
 from o2a.utils.el_utils import normalize_path
 
 
 class PrepareMixin:
     """Mixin used to add Prepare node capability to a node"""
 
-    @staticmethod
-    def has_prepare(oozie_node):
-        return bool(xml_utils.find_nodes_by_tag(oozie_node, "prepare"))
+    def __init__(self, oozie_node):
+        self.oozie_node_for_prepare = oozie_node
 
-    def get_prepare_command(self, oozie_node: ET.Element, params: Dict[str, str]):
-        # In BashOperator in Composer we can't read from $DAGS_FOLDER (~/dags) - permission denied.
-        # However we can read from ~/data -> /home/airflow/gcs/data.
-        # The easiest way to access it is using the $DAGS_FOLDER env variable.
-        delete_paths, mkdir_paths = self.parse_prepare_node(oozie_node, params)
-        if delete_paths or mkdir_paths:
+    def has_prepare(self):
+        prepare_nodes = xml_utils.find_nodes_by_tag(self.oozie_node_for_prepare, "prepare")
```

Comment:
````
<xs:element name="prepare" type="workflow:PREPARE" minOccurs="0" maxOccurs="1"/>
````
I think, you should use `find_node_by_tag`.

Reply:
Indeed. a bit simpler. Fixed in 635479a79f2413a4a0298e4878d062c9b4d72ccf